sample/lib/build.gradle.kts

```diff
@@ -24,10 +24,14 @@ android {
     namespace = "com.spotify.ruler.sample.lib"
     compileSdk = 33
     defaultConfig {
-        minSdk = 23
+        minSdk = 21
         targetSdk = 33
     }
     lint {
         warningsAsErrors = true
     }
 }
+
+dependencies {
+    implementation("androidx.constraintlayout:constraintlayout:2.1.4")
```

Comment:
Nit - I'd stay consistent here and manage this in the `ruler/buildSrc/main/kotlin/Dependencies.kt` file instead of hardcoding here.